WordPress/src/main/res/values/colors.xml

```diff
@@ -309,4 +315,12 @@
     <color name="reader_following_empty_view_outline">@color/gray_10</color>
     <color name="reader_following_empty_view_middle_plus">@android:color/white</color>
     <color name="reader_following_empty_view_background">@android:color/white</color>
+
+    <!--Reader Post Card -->
+    <color name="reader_post_list_background">@color/neutral_0</color>
+    <color name="reader_card_background">@color/white</color>
+    <color name="reader_label_text">#6c7781</color>
```

Comment:
@ashiagr Are the reader_label_text and reader_image_featured_border are not in the color palette (as you made reference in the notes)?

Reply:
Yes, I couldn't find in the color palette. I've to discuss it with Shakti.